src/server/session.ts

```diff
@@ -1411,15 +1417,72 @@ namespace ts.server {
             return ts.getSupportedCodeFixes();
         }
 
+        private extractStartAndEndPositionFromTextRangeRequestArgs(args: protocol.TextRangeRequestArgs, scriptInfo: ScriptInfo): { startPosition: number, endPosition: number } {
+            const startPosition = getStartPosition();
+            const endPosition = getEndPosition();
+            return { startPosition, endPosition };
+
+            function getStartPosition() {
+                return args.startPosition !== undefined ? args.startPosition : scriptInfo.lineOffsetToPosition(args.startLine, args.startOffset);

```

Comment:
Do you want to save the result in `args.startPosition` (resp. `args.endPosition`)?